commons/c2cgeoportal_commons/models/main.py

```diff
@@ -294,22 +296,32 @@ class LayergroupTreeitem(Base):
     ]
 
     # required by formalchemy
-    id = Column(Integer, primary_key=True)
-    description = Column(Unicode)
-    treegroup_id = Column(Integer, ForeignKey(_schema + '.treegroup.id'))
+    id = Column(Integer, primary_key=True, info={
+        'colanderalchemy': { 'widget': HiddenWidget() } }
+    )
+    description = Column(Unicode, info={ 'colanderalchemy': { 'exclude': True }})
```

Comment:
Why excluded?

Reply:
This description column in group / item relation seems to be an error.
Why do we want a description in a link between two nodes ?
How do you want to present this to the user ?
Note that the treeitem already have a description.
Am I missing something ?

Reply:
I don't catch that's in the relation, a customer need this for special operations, then we can't remove it, but effectively it's clear that she is hidden in the new interface :-)